frontend/src/lib/apiClient.js

```diff
@@ -9,7 +9,10 @@ const apiClient = axios.create({
   baseURL,
 });
 
-// AUTHENTICATION
```

Comment:
Can we re-sort the functions in here into their categories and replace the category comments? It's hellishly hard to parse when the user and recipe and login functions are all jumbled together.